devicetypes/shinasys/sihas-zigbee-button.src/sihas-zigbee-button.groovy

```diff
@@ -0,0 +1,186 @@
+/*
+ *  Copyright 2021 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+    definition (name: "SiHAS Zigbee Button", namespace: "shinasys", author: "SHINA SYSTEM", mnmn: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: true ) {
```

Comment:
it looks like the only difference between this handler and the zigbee-button DTH is the voltage range. 

you could just add your fingerprints and some conditional logic around the voltage report handling instead of creating a new DTH

Reply:
First, I will cancel this file(sihas-zigbee-button.groovy) and proceed later.